rio_tiler/io/rasterio.py

```diff
@@ -515,23 +517,24 @@ def feature(
             height (int, optional): Output height of the array.
             width (int, optional): Output width of the array.
             buffer (int or float, optional): Buffer on each side of the given aoi. It must be a multiple of `0.5`. Output **image size** will be expanded to `output imagesize + 2 * buffer` (e.g 0.5 = 257x257, 1.0 = 258x258).
+            all_touched (bool, optional): Shape rasterization parameter. If True, all pixels touched the input shape will be burned in. If false, only pixels whose center is within the input shape or that are selected by Bresenham's line algorithm will be burned in.
```

Comment:
The more I read this the more I think we should always set this to true 🤷

Reply:
That's fine with me. This parameter has an impact on features that are small or thin when compared with a single pixel. I.e. linear features or small areas. So for masking its usually not a big difference either way.

Reply:
I kept this in the latest version, but let me know if you prefer to remove it and default to `True`. Either way is fine for me.